app/src/main/java/org/eyeseetea/malariacare/domain/entity/Form.java

```diff
@@ -1,22 +1,23 @@
 package org.eyeseetea.malariacare.domain.entity;
 
 
+import static org.eyeseetea.malariacare.domain.utils.RequiredChecker.required;
+
 public class Form {
     private long id;
     private String name;
     private int index;
     private Type type;
     private Country country;
 
-    public Form() {
-    }
+    public Form(long id, String name, int index, Type type,
+            Country country) {
 
-    private Form(Builder builder) {
```

Comment:
Can you remove private setters? It's not necessary we have private setters, so we reduce the size of the class